docs/user_guide/remote_processing.rst

```diff
@@ -7,98 +7,63 @@ The SiliconCompiler project supports a remote processing model that leverages th
-
-.. list-table::
-   :widths: 10 10
-   :header-rows: 1
-
-   * - Keypath
-     - Restrictions
-
-   * - ``design``
-     - Must only contain alphanumeric characters or underscores
-   * - ``jobname``
-     - Must only contain alphanumeric characters or underscores
-   * - ``dir``
-     - Must only contain alphanumeric characters or underscores
-   * - ``constraint``
-     - Only allowed for fpgaflow
-   * - ``target``
-     - Must be one of ``asicflow_freepdk45``, ``asicflow_skywater130``, or ``fpgaflow_ice40up5k-sg48``
-   * - ``library, ...``
-     - Allowed if key isn't the name of a built-in PDK standard cell library
-   * - ``source``
-     - None
-   * - ``testbench``
-     - None
-   * - ``waveform``
-     - None
-   * - ``clock, ...``
-     - None
-   * - ``supply, ...``
-     - None
-   * - ``param, ...``
-     - None
-   * - ``define``
-     - None
-   * - ``ydir``
-     - None
-   * - ``idir``
-     - None
-   * - ``vlib``
-     - None
-   * - ``libext``
-     - None
-   * - ``cmdfile``
-     - None
-   * - ``activityfactor``
-     - None
-   * - ``read, spef, ...``
-     - None
-   * - ``read, sdf, ...``
-     - None
-   * - ``read, saif, ...``
-     - None
-   * - ``read, gds, ...``
-     - None
-   * - ``read, def, ...``
-     - None
-   * - ``read, netlist, ...``
-     - None
-   * - ``read, pcf, ...``
-     - None
-   * - ``read, vcd, ...``
-     - None
-   * - ``asic, ...``
-     - None
-   * - ``metric, ...``
-     - None
-   * - ``fpga, ...``
-     - None
-   * - ``flowarg, ...``
-     - None
-   * - ``techarg, ...``
-     - None
-   * - ``clean``
-     - None
-   * - ``loglevel``
-     - None
-   * - ``novercheck``
-     - None
-   * - ``checkonly``
-     - None
-   * - ``eda, ..., variable, ...``
-     - None
+.. note::
+
+    Note that our public beta currently only supports open-source tools and PDKs. You can access the public beta without a signup or login, and it is designed to delete your data after your jobs finish, but it is not intended to process proprietary or restricted intellectual property! Please `review our terms of service <https://www.siliconcompiler.com/terms-of-service>`_, and do not submit IP which you are not allowed to distribute.
+
+    Currently, our public beta servers will only return a report containing a rendering and metrics for your build results. For the full GDS results, you can build and install the open-source tools and run the job on your local machine. We provide `setup scripts to help install these tools on Ubuntu systems <https://github.com/siliconcompiler/siliconcompiler/tree/main/setup>`_.
```

Comment:
I would link back to the install instructions again here for a local run since it talks about minimum tools required to be installed for a run and also provides caveat that the setup scripts are not end-all-be-all install instructions.

> you can build and install the open-source tools and run the job on your local machine. **See :ref:`local run` for instructions.**